packages/bpk-component-map/src/BpkIconMarkerBackground.module.scss

```diff
@@ -42,8 +42,11 @@
   }
 
   &--selected {
-    fill: $bpk-surface-default-day;
-
+    @include bpk-themeable-property(
+      fill,
+      --bpk-icon-marker-default-selected-fill-color,
+      $bpk-surface-default-day
+    );
```

Comment:
We won't need to make this a themable colour and `bpk-surface-default-day` will be fine :)